src/actions/index.js

```diff
@@ -9,10 +10,23 @@ export const DECREMENT = 'DECREMENT';
 // logic of updating the central Redux store. That
 // is left to the reducer(s).
 
-export const increment = () => {
-  // Fill in this function
+export const increment = num => {
+  return {
+    type: INCREMENT,
+    payload: num
+  }
```

Comment:
this is perfectly functional and isn't wrong, per se -- just be aware that since the reducer functions already have access to the state.count value in their file, it isn't necessary to pass the number in or send a payload at all